conf.py

```diff
@@ -505,12 +505,17 @@
 # -- User Defined Methods ------------------------------------------------
 sys.path.append(os.getcwd())
 
+if sys.platform == "win32":
```

Comment:
I think the issue here might be that @dccowan has the wrong anaconda installed. Might be worth uninstalling and ensuring that the 32 bit is installed